text_formatter_js.go

```diff
@@ -0,0 +1,11 @@
+// +build js
```

Comment:
Can we keep the same naming convention ? This file could be named terminal_check_js.go